src/net/sourceforge/kolmafia/request/GenericRequest.java

```diff
@@ -1394,11 +1394,11 @@ public void externalExecute() {
   public static final boolean shouldIgnore(final GenericRequest request) {
     String requestURL = GenericRequest.decodeField(request.formURLString);
     return requestURL == null
-        ||
+        //        ||
```

Comment:
Absolutely not. Remove your private change. :)

Reply:
Luckily this will cause a test I added to fail

Reply:
usually I don't make this mistake... thank you for adding that test, Sam :)